packages/tsconfig/base.json

```diff
@@ -9,13 +9,14 @@
     "forceConsistentCasingInFileNames": true,
     "inlineSources": false,
     "isolatedModules": true,
+    "moduleResolution": "node",
     "noUnusedLocals": false,
     "noUnusedParameters": false,
     "preserveWatchOutput": true,
-    "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "ES2022"
+    "target": "ES2022",
+    "resolveJsonModule": true
```

Comment:
@I would prefer to have these things in alphabetical order. This way you can easily understand additions in the future.